service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/SendEhrExtractCoreTaskDispatcher.java

```diff
@@ -29,5 +28,8 @@ public void send(EhrExtractStatus ehrExtractStatus) {
             .build();
 
         taskDispatcher.createTask(sendEhrExtractCoreTaskDefinition);
+        LOGGER.info(String.format("SendEhrExtractCoreTaskDefinition added to task queue, conversationId: %s",
```

Comment:
Don't explicitly log conversationId. This is part of our logback.xml format and it always included.